jvb/src/test/kotlin/org/jitsi/videobridge/ConferenceTest.kt

```diff
@@ -16,74 +16,55 @@
-        every { get() } returns octoRelayService
-    }
-
-    private val bundleContext = mockk<BundleContext> {
-        every { registerService(any() as String, any(), any()) } returns null
-    }
-    private val videobridge = mockk<Videobridge> {
-        every { bundleContext } returns this@ConferenceTest.bundleContext
-    }
+    // The octo relay binds on a port
+    override fun isolationMode(): IsolationMode? = IsolationMode.SingleInstance
 
-    init {
-        mockkStatic("org.jitsi.videobridge.octo.OctoRelayServiceProviderKt")
-        every { singleton() } returns octoRelaySupplier
-    }
+    private val videobridge = mockk<Videobridge>()
 
     init {
         val name = JidCreate.entityBareFrom("roomName@somedomain.com")
+        withNewConfig(newConfigOctoEnabled, loadDefaults = true) {
+            octoRelayServiceProvider().get()?.start()
+        }
+
         context("Adding local endpoints should work") {
-            withNewConfig(newConfigOctoEnabled, loadDefaults = true) {
-                with(Conference(videobridge, "id", name, false, Conference.GID_NOT_SET)) {
-                    endpointCount shouldBe 0
-                    createLocalEndpoint("abcdabcd", true)
-                    endpointCount shouldBe 1
-                    debugState.shouldBeValidJson()
-                }
+            with(Conference(videobridge, "id", name, false, Conference.GID_NOT_SET)) {
+                endpointCount shouldBe 0
+                createLocalEndpoint("abcdabcd", true)
+                endpointCount shouldBe 1
+                debugState.shouldBeValidJson()
             }
         }
         context("Enabling octo should fail when the GID is not set") {
-            withNewConfig(newConfigOctoEnabled, loadDefaults = true) {
-                with(Conference(videobridge, "id", name, false, Conference.GID_NOT_SET)) {
-                    isOctoEnabled shouldBe false
-                    shouldThrow<IllegalStateException> {
-                        tentacle
-                    }
-                    debugState.shouldBeValidJson()
+            with(Conference(videobridge, "id", name, false, Conference.GID_NOT_SET)) {
+                isOctoEnabled shouldBe false
+                shouldThrow<IllegalStateException> {
+                    tentacle
                 }
+                debugState.shouldBeValidJson()
             }
         }
         context("Enabling octo should work") {
-            withNewConfig(newConfigOctoEnabled, loadDefaults = true) {
-                octoRelayService.start()
-                with(Conference(videobridge, "id", name, false, 1234)) {
-                    isOctoEnabled shouldBe false
-                    tentacle
-                    isOctoEnabled shouldBe true
-                    tentacle.setRelays(listOf("127.0.0.1:4097"))
+            with(Conference(videobridge, "id", name, false, 1234)) {
+                isOctoEnabled shouldBe false
+                tentacle
```

Comment:
Not for this PR, but reading this test flow makes me wonder if the API we have set up in Conference makes sense (`isOctoEnabled` checks if `tentacle != null`, but, even if Octo _is_/_will be_ enabled, it won't be reflected until `getTentacle` has been called)